### ISSUE TITLE
Error when running tango info

### ISSUE THREAD

**Issue by schmmd** [allenai/tango#338]
### 🐛 Describe the bug

I installed tango via `pip install ai2-tango`.

```
$ tango info                                                                                               
Tango version 0.3.6 (python 3.9.12)

Integrations:
OMP: Error #15: Initializing libiomp5.dylib, but found libomp.dylib already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
fish: 'tango info' terminated by signal SIGABRT (Abort)
```

### Versions

absl-py @ file:///home/conda/feedstock_root/build_artifacts/absl-py_1654189308200/work
ai2-tango @ file:///Users/runner/miniforge3/conda-bld/tango-split_1638944535621/work
aiohttp @ file:///Users/runner/miniforge3/conda-bld/aiohttp_1649013201663/work
aiosignal @ file:///home/conda/feedstock_root/build_artifacts/aiosignal_1636093929600/work
async-timeout @ file:///home/conda/feedstock_root/build_artifacts/async-timeout_1640026696943/work
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1640799537051/work
base58 @ file:///home/conda/feedstock_root/build_artifacts/base58_1635724186165/work
blinker==1.4
boto3 @ file:///home/conda/feedstock_root/build_artifacts/boto3_1657783699883/work
botocore @ file:///home/conda/feedstock_root/build_artifacts/botocore_1657756950790/work
brotlipy @ file:///Users/runner/miniforge3/conda-bld/brotlipy_1648854211613/work
cached-path==0.3.2
cachetools @ file:///home/conda/feedstock_root/build_artifacts/cachetools_1640686991047/work
certifi==2022.6.15
cffi @ file:///Users/runner/miniforge3/conda-bld/cffi_1625835329396/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1655906222726/work
click @ file:///Users/runner/miniforge3/conda-bld/click_1651215220522/work
click-help-colors @ file:///home/conda/feedstock_root/build_artifacts/click-help-colors_1638331082124/work
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1655412516417/work
cryptography @ file:///Users/runner/miniforge3/conda-bld/cryptography_1657174062636/work
dataclasses @ file:///home/conda/feedstock_root/build_artifacts/dataclasses_1628958434797/work
datasets @ file:///home/conda/feedstock_root/build_artifacts/datasets_1635906567859/work
dill @ file:///home/conda/feedstock_root/build_artifacts/dill_1653058582944/work
docker-pycreds==0.4.0
filelock==3.3.2
frozenlist @ file:///Users/runner/miniforge3/conda-bld/frozenlist_1648771747683/work
fsspec @ file:///home/conda/feedstock_root/build_artifacts/fsspec_1653010523205/work
future @ file:///Users/runner/miniforge3/conda-bld/future_1649010190079/work
gitdb @ file:///home/conda/feedstock_root/build_artifacts/gitdb_1635085722655/work
GitPython @ file:///home/conda/feedstock_root/build_artifacts/gitpython_1645531658201/work
google-api-core @ file:///home/conda/feedstock_root/build_artifacts/google-api-core-split_1655668757136/work
google-auth @ file:///home/conda/feedstock_root/build_artifacts/google-auth_1657692258949/work
google-auth-oauthlib @ file:///home/conda/feedstock_root/build_artifacts/google-auth-oauthlib_1630497468950/work
google-cloud-core @ file:///home/conda/feedstock_root/build_artifacts/google-cloud-core_1654835241407/work
google-cloud-storage @ file:///home/conda/feedstock_root/build_artifacts/google-cloud-storage_1638482791995/work
google-crc32c @ file:///Users/runner/miniforge3/conda-bld/google-crc32c_1651517262819/work
google-resumable-media @ file:///home/conda/feedstock_root/build_artifacts/google-resumable-media_1654213463347/work
googleapis-common-protos @ file:///Users/runner/miniforge3/conda-bld/googleapis-common-protos-feedstock_1657690463235/work
grpcio @ file:///Users/runner/miniforge3/conda-bld/grpcio_1645230591110/work
huggingface-hub @ file:///home/conda/feedstock_root/build_artifacts/huggingface_hub_1636414419863/work
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1642433548627/work
importlib-metadata @ file:///Users/runner/miniforge3/conda-bld/importlib-metadata_1653252883293/work
jmespath @ file:///home/conda/feedstock_root/build_artifacts/jmespath_1655568249366/work
jsonnet @ file:///Users/runner/miniforge3/conda-bld/jsonnet_1636234966467/work
Markdown @ file:///home/conda/feedstock_root/build_artifacts/markdown_1651821407140/work
mkl-fft==1.3.1
mkl-random==1.2.2
mkl-service==2.4.0
more-itertools @ file:///home/conda/feedstock_root/build_artifacts/more-itertools_1652515549581/work
multidict @ file:///Users/runner/miniforge3/conda-bld/multidict_1648882472994/work
multiprocess @ file:///Users/runner/miniforge3/conda-bld/multiprocess_1653595773546/work
numpy @ file:///private/var/folders/sy/f16zz6x50xz3113nwtb9bvq00000gp/T/abs_e3f85f54-0572-4c0e-b190-4bc4766fc3fenewxarhq/croots/recipe/numpy_and_numpy_base_1652801682879/work
oauthlib @ file:///home/conda/feedstock_root/build_artifacts/oauthlib_1643507977997/work
overrides @ file:///home/conda/feedstock_root/build_artifacts/overrides_1592431494961/work
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1637239678211/work
pandas==1.4.1
pathtools==0.1.2
promise @ file:///Users/runner/miniforge3/conda-bld/promise_1649695782852/work
protobuf==3.19.4
psutil @ file:///Users/runner/miniforge3/conda-bld/psutil_1653089238227/work
pyarrow==6.0.1
pyasn1==0.4.8
pyasn1-modules==0.2.7
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1636257122734/work
pyDeprecate @ file:///home/conda/feedstock_root/build_artifacts/pydeprecate_1622496226936/work
PyJWT @ file:///home/conda/feedstock_root/build_artifacts/pyjwt_1652398519695/work
pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1643496850550/work
pyparsing @ file:///home/conda/feedstock_root/build_artifacts/pyparsing_1652235407899/work
PySocks @ file:///Users/runner/miniforge3/conda-bld/pysocks_1648857336012/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/python-dateutil_1626286286081/work
pytorch-lightning @ file:///home/conda/feedstock_root/build_artifacts/pytorch-lightning_1633020752766/work
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1647961439546/work
pyu2f @ file:///home/conda/feedstock_root/build_artifacts/pyu2f_1604248910016/work
PyYAML @ file:///Users/runner/miniforge3/conda-bld/pyyaml_1648757420486/work
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1656534056640/work
requests-oauthlib @ file:///home/conda/feedstock_root/build_artifacts/requests-oauthlib_1643557462909/work
rsa @ file:///home/conda/feedstock_root/build_artifacts/rsa_1637781155505/work
s3transfer @ file:///home/conda/feedstock_root/build_artifacts/s3transfer_1654039987929/work
sentry-sdk @ file:///home/conda/feedstock_root/build_artifacts/sentry-sdk_1657735318067/work
setproctitle @ file:///Users/runner/miniforge3/conda-bld/setproctitle_1650296537432/work
shortuuid @ file:///home/conda/feedstock_root/build_artifacts/shortuuid_1656529349686/work
six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
smmap @ file:///home/conda/feedstock_root/build_artifacts/smmap_1611376390914/work
tensorboard @ file:///home/conda/feedstock_root/build_artifacts/tensorboard_1655042669710/work/tensorboard-2.9.1-py3-none-any.whl
tensorboard-data-server @ file:///Users/runner/miniforge3/conda-bld/tensorboard-data-server_1649932875354/work/tensorboard_data_server-0.6.0-py3-none-macosx_10_9_x86_64.whl
tensorboard-plugin-wit @ file:///home/conda/feedstock_root/build_artifacts/tensorboard-plugin-wit_1641458951060/work/tensorboard_plugin_wit-1.8.1-py3-none-any.whl
torch==1.10.2
torchmetrics @ file:///home/conda/feedstock_root/build_artifacts/torchmetrics_1656568368092/work
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1632160078689/work
typing_extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1656706066251/work
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1657224465922/work
wandb @ file:///home/conda/feedstock_root/build_artifacts/wandb_1657282472508/work
Werkzeug @ file:///home/conda/feedstock_root/build_artifacts/werkzeug_1651670883478/work
xxhash @ file:///Users/runner/miniforge3/conda-bld/python-xxhash_1649442463882/work
yarl @ file:///Users/runner/miniforge3/conda-bld/yarl_1648966566248/work
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1649012893348/work

**Comment by epwalsh** [comment]
That's a really old version of Tango.. can you try on the latest?

**Comment by schmmd** [comment]
I deleted conda, all my environments, reinstalled miniconda, and tried again and got the same error.  So I'm pretty sure our conda installation instructions do not work, and they are listed first in the readme.

**Comment by schmmd** [comment]
This worked when I installed via pip:

```
tango info                                                                                                     (tango) 
Tango version 0.10.0 (python 3.9.12)                                                                                                             
                                                                                                                                                 
Integrations:                                                                                                                                    
 ✓ fairscale                                                                                                                                     
 ✓ datasets                                                                                                                                      
 ✓ torch                                                                                                                                         
 ✓ beaker                                                                                                                                        
 ✓ transformers                                                                                                                                  
 ✓ pytorch_lightning                                                                                                                             
 ✓ wandb 
 ```

**Comment by izhx** [comment]
Can we run tango with only `torch` and `transformers`? 
I installed tango via `pip install ai2-tango[transformers]`. 
But 

```
Tango version 1.0.0 (python 3.8.13)

Integrations:
 ✓ torch
[10/09/22 15:25:15] ERROR    Uncaught exception
Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/wandb/__init__.py", line 27, in <module>
    import wandb
ModuleNotFoundError: No module named 'wandb'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/tango/bin/tango", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/__main__.py", line 481, in info
    import_module_and_submodules(integration)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/util.py", line 104, in import_module_and_submodules
    module = importlib.import_module(package_name)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/wandb/__init__.py", line 29, in <module>
    raise IntegrationMissingError("wandb")
tango.common.exceptions.IntegrationMissingError: 'wandb' integration can't be used due to missing dependencies (wandb)
```

I installed **ALL** Integrations (wandb, jax, fairscale, etc), and then `tango info` worked.

```
Tango version 1.0.0 (python 3.8.13)

Integrations:
 ✓ torch
 ✗ wandb (not installed)
 ✗ beaker (not installed)
 ✗ flax (not installed)
 ✓ fairscale
 ✓ datasets
 ✓ transformers
 ✓ pytorch_lightning
```

However, I still can not run `tango run config/crf.jsonnet -d workspace/ -i src`,
```
(tango) ubuntu@lthpc:~/hill-ner$ tango run config/crf.jsonnet -d workspace/ -i src
usage: tango [-h] [--config CONFIG] [--cuda CUDA] [--name NAME] [--node-rank NODE_RANK] [--file-friendly-logging] [-f] [--seed SEED] [--batch-size BATCH_SIZE]
tango: error: unrecognized arguments: run config/crf.jsonnet -d workspace/ -i src
[10/09/22 16:23:57] ERROR    Uncaught exception
Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/tango/bin/tango", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/__main__.py", line 321, in run
    _run(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/__main__.py", line 774, in _run
    step_graph = StepGraph.from_params(params.pop("steps"))
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step_graph.py", line 126, in from_params
    step_dict[step_name] = Step.from_params(step_params, step_name=step_name)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step.py", line 376, in from_params
    constructed_arg = pop_and_construct_arg(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 352, in pop_and_construct_arg
    return construct_arg(class_name, name, popped_params, annotation, default)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 412, in construct_arg
    return construct_arg(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 456, in construct_arg
    result = annotation.from_params(popped_params)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step.py", line 342, in from_params
    as_registrable.search_modules(params["type"])
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/registrable.py", line 251, in search_modules
    try_import(module)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/registrable.py", line 193, in try_import
    import_module_and_submodules(module)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/util.py", line 115, in import_module_and_submodules
    import_module_and_submodules(subpackage, exclude=exclude)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/util.py", line 104, in import_module_and_submodules
    module = importlib.import_module(package_name)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/wandb/flax_train_callback.py", line 8, in <module>
    from tango.integrations.flax.train_callback import TrainCallback
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/flax/__init__.py", line 26, in <module>
    from .format import FlaxFormat
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/flax/format.py", line 4, in <module>
    from flax.training import checkpoints
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/flax/training/checkpoints.py", line 38, in <module>
    from tensorflow import errors as tf_errors
ModuleNotFoundError: No module named 'tensorflow'
```

And I installed `tensorflow`, but still not worked

```
(tango) ubuntu@lthpc:~/hill-ner$ tango run config/crf.jsonnet -d workspace/ -i src
2022-10-09 16:31:20.366615: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-10-09 16:31:20.487803: I tensorflow/core/util/util.cc:169] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2022-10-09 16:31:20.519678: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2022-10-09 16:31:22.045251: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory
2022-10-09 16:31:22.045343: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory
2022-10-09 16:31:22.045353: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
usage: tango [-h] [--config CONFIG] [--cuda CUDA] [--name NAME] [--node-rank NODE_RANK] [--file-friendly-logging] [-f] [--seed SEED] [--batch-size BATCH_SIZE]
tango: error: unrecognized arguments: run config/crf.jsonnet -d workspace/ -i src
[10/09/22 16:31:24] ERROR    Uncaught exception
Traceback (most recent call last):
  File "/home/ubuntu/.conda/envs/tango/bin/tango", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/__main__.py", line 321, in run
    _run(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/__main__.py", line 774, in _run
    step_graph = StepGraph.from_params(params.pop("steps"))
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step_graph.py", line 126, in from_params
    step_dict[step_name] = Step.from_params(step_params, step_name=step_name)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step.py", line 376, in from_params
    constructed_arg = pop_and_construct_arg(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 352, in pop_and_construct_arg
    return construct_arg(class_name, name, popped_params, annotation, default)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 412, in construct_arg
    return construct_arg(
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/from_params.py", line 456, in construct_arg
    result = annotation.from_params(popped_params)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/step.py", line 342, in from_params
    as_registrable.search_modules(params["type"])
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/registrable.py", line 251, in search_modules
    try_import(module)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/registrable.py", line 193, in try_import
    import_module_and_submodules(module)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/common/util.py", line 104, in import_module_and_submodules
    module = importlib.import_module(package_name)
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/beaker/__init__.py", line 17, in <module>
    from .executor import (
  File "/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/tango/integrations/beaker/executor.py", line 11, in <module>
    from beaker import (
ImportError: cannot import name 'Beaker' from 'beaker' (/home/ubuntu/.conda/envs/tango/lib/python3.8/site-packages/beaker/__init__.py)
```

Finally, I created a new env and run `pip install ai2-tango[all]`, it worked.

But I don't need them 😭 ...

**Comment by epwalsh** [comment]
@izhx oof, sorry about that! Looks like ran into several different issues. I think I've fixed them all in #447

**Comment by dirkgr** [comment]
This is fixed then?
